### ISSUE TITLE
Conv Bn ReLU pattern

### ISSUE THREAD

**Issue by romulus0914** [google/automl#30]
Hi,

I would like to ask about the pattern of convolution, batch normalization, and relu for feature fusion. In the paper, it is suggested the 'conv-bn-relu' pattern, however, in this repo, the hyper-parameter config is set to be 'relu-conv-bn' pattern. Which one is more preferred?

Best regards.
